float32[] des_eep
float32 duration
---